utils/data-ingestor.js

```diff
@@ -0,0 +1,229 @@
+const { find } = require('lodash');
+const df = require('dataframe-js');
+
+const specialCalcConfigs = require('../special-calculations');
+const calculateMedianError = require('../utils/calculate-median-error');
+const interpolate = require('../utils/interpolate');
+const { executeWithData: executeFormula } = require('../utils/formula');
+
+const {
+  INFLATION_FACTOR,
+  RENAME_COLS,
+  PREV_YEAR,
+  CUR_YEAR,
+} = require('../special-calculations/data/constants');
+
+/*
+ * The DataIngester class is used to convert profile data from postgres query into a Dataframe.
+ * Additionally for aggregate profiles, the DataIngester recalculates 'special' variables, configured
+ * in 'special-calculations/[profile].js'.
+ * NOTE: Dataframes (https://gmousse.gitbooks.io/dataframe-js/#dataframe-js) are immutable, so they cannot be modified
+ * in place. That is why all functions mutating a Dataframe also return one.
+ */
+class DataIngester {
+  /*
+   * Creates a DataIngester
```

Comment:
nit: the file is named `data-ingestor` but the class and comment refer to `data-ingester` - could make it hard to grep things later?

Reply:
def, will fix the typo